widgetssdk/src/test/java/com/glia/widgets/chat/ChatManagerTest.kt

```diff
@@ -610,24 +609,24 @@ class ChatManagerTest {
         val chatMessageInternal = mockChatMessage<SystemMessage>()
         doReturn(true).whenever(state).isNew(chatMessageInternal)
 
-        whenever(loadHistoryUseCase()) doReturn Single.just(mock())
+        whenever(loadHistoryUseCase()) doReturn Single.just(ChatHistoryResponse(emptyList()))
         whenever(onMessageUseCase()) doReturn Observable.just(chatMessageInternal)
 
-        val chatItems = mutableListOf<ChatItem>(mock())
         subjectUnderTest.apply {
             val onHistoryLoaded = mock<(hasHistory: Boolean) -> Unit>()
             val onQuickReplyReceived = mock<(List<GvaButton>) -> Unit>()
             val onOperatorMessageReceived = mock<(count: Int) -> Unit>()
-            val itemsFlowable = initialize(onHistoryLoaded, onQuickReplyReceived, onOperatorMessageReceived)
-            verify(this).subscribe(onHistoryLoaded, onOperatorMessageReceived, onQuickReplyReceived)
 
-            val test = itemsFlowable.test()
+            initialize(onHistoryLoaded, onQuickReplyReceived, onOperatorMessageReceived).test().assertNoErrors().awaitCount(1)
 
-            stateProcessor.onNext(ChatManager.State(chatItems = chatItems))
+            verify(this).subscribe(onHistoryLoaded, onOperatorMessageReceived, onQuickReplyReceived)
+            verify(this).subscribeToState(onHistoryLoaded, onOperatorMessageReceived)
+            verify(this).subscribeToQuickReplies(onQuickReplyReceived)
 
             verify(this, atLeastOnce()).updateQuickReplies(any())
 
-            assertEquals(test.values().last().last(), chatItems.last())
+            stateProcessor.onNext(state)
+            verify(state).immutableChatItems
         }
```

Comment:
Just for understanding. This test and changes are not directly related to the ticket, right?

Reply:
Yes, this test just started to fail in bitrise, so I had to fix it.